### PR TITLE
[16.0][FIX] search_engine_image_thumbnail: Fix size lookup

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

**Comment by lmignon** [comment]
cc @sebastienbeau @qgroulard @marielejeune @sbejaoui

**Review by sbejaoui — APPROVED** [review]
Thanks @lmignon

File: search_engine_image_thumbnail/models/se_index.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ class SeIndex(models.Model):
     image_field_thumbnail_size_ids = fields.One2many(
         string="Thumbnail Sizes",
         comodel_name="se.image.field.thumbnail.size",
-        compute="_compute_image_field_thumbnail_size_ids",
```
Comment (qgroulard) [inline search_engine_image_thumbnail/models/se_index.py:14]: The function `_compute_image_field_thumbnail_size_ids` can be removed then.

**Review by qgroulard — APPROVED** [review]

**Comment by sbidoul** [comment]
@lmignon can you retry?

**Comment by lmignon** [comment]
> @lmignon can you retry?

It works. Thank you.

**Pull request merged**